extensions/amp-ad/0.1/amp-ad.css

```diff
@@ -48,9 +48,13 @@ amp-embed iframe {
   padding: 2px 4px;
 }
 
-amp-ad[type=adsense] > iframe {
+amp-ad[data-a4a-upgrade-type] > iframe {
```

Comment:
Yeah...  I guess we don't right now.  Bah.  This commits us to keeping this attribute around.  Erm...  In your tests for this code, can you add a check that the CSS is applied to the correct slots?  (Not sure if there's a way to test that?)  If so, you could put that in a test along with a note that the code depends on the existence of the `data-a4a-upgrade-type` attribute and that if this test breaks here, it's probably that the attribute got fiddled somehow?  That will at least trigger a clear test failure if we ever forget and delete that attribute.